app/utils/EitherSupport.scala

```diff
@@ -0,0 +1,12 @@
+package utils
+
+object EitherSupport {
```

Comment:
It would be great to have a short comment here explaining things

Reply:
scaladoc API comment, along with implementation specific comment added.